### PR TITLE
restructure code to make it more organized by using modules.

### PR DESCRIPTION
**Pull request by admirerbrown**
in this project i did the following activities:
- installed linters 
- ensured project is error free 
- refactored code 
- Display the current date below the navigation bar.

- did the following project requirement:
- Display the current date below the navigation bar.
For this requirement you should use a library [Luxon](https://moment.github.io/luxon/#/install?id=es6)
Download the script and save it to your project.
Import it as a module.

### CODE REVIEW

File: modules/ui_controller.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+function pageFunctionality() {
```
Comment (julie-ify) [inline modules/ui_controller.js]: - [x] Kindly use arrow syntax for this function.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+document.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const title = document.getElementById('title').value;
+  const authorInput = document.getElementById('author').value;
+
+  const newBook = new Book.Book(title, authorInput); // create new book object
+  bookList.push(newBook);
+
+  // send book to local storage
+  newBook.storeBook(bookList);
```
Comment (julie-ify) [inline index.js]: - [x] **[OPTIONAL]** For a better user experience I think it would be nice to clear the inputs after successfully adding a book.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @admirerbrown,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — APPROVED** [review]
# Status: APPROVED 🟢🟢🟢 

Hi @admirerbrown,

Good job on making the changes asked by the previous reviewer 👍 
Your project is complete!  

## Highlights:
✅ You used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ Used ES6 and arrow function syntax as required 👍 
✅ Adding and removing a book features work fine👍

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**